context/AuthContextProvider.test.tsx

```diff
@@ -55,11 +63,36 @@ describe('AuthContextProvider', () => {
 
     mockCreateEmailPasswordSession.mockRejectedValue(mockError);
 
-    const error = await loginAccount({user, router, getUser});
+    const error = await loginAccount({ user, router, getUser });
+
+    expect(error).toEqual(mockError);
+    expect(toast.custom).toHaveBeenCalledWith(
+      <Alert variant={AlertVariants.Error} message="Something went wrong!" />,
+    );
+  });
+
+  //logout tests
+  test('after a successful logout it shows success notification', async () => {
```

Comment:
There seems to be some inconsistency with double and single quotes here. Please update this so it uses one or the other.